airbyte-config-oss/specs-oss/src/main/java/io/airbyte/configoss/specs/ConnectorMarkdownGenerator.java

```diff
@@ -102,11 +104,17 @@ private String buildMarkdownTable(final List<JsonNode> definitions, final String
       final String dockerImage = definition.get("dockerRepository").asText() + ":" + definition.get("dockerImageTag").asText();
       final String releaseStage = definition.get("releaseStage") != null ? definition.get("releaseStage").asText() : "unknown";
       final String documentationUrl = definition.get("documentationUrl") != null ? definition.get("documentationUrl").asText() : "";
-      final String docLink = !documentationUrl.equals("") ? "[link](" + documentationUrl + ")" : "missing";
-      final String codeLink = "[code](" + githubCodeBase + "/" + codeName + ")";
+      final String docLink = !documentationUrl.equals("") ? "[docs](" + documentationUrl + ")" : "missing";
+      final String issuesLabel = "connectors/" + (codeName.contains("source-") ? "source" : "destination") + "/"
```

Comment:
On a scale of 1-10 on the hurt my brain to read scale this felt about a 4.

Any chance we can make that a named function or add a comment?

Reply:
well... what I really hope is that this whole file goes away and #metadata makes this file soon...

Reply:
comment commented!